src/Symfony/Bridge/Twig/Extension/SerializeRuntime.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\Twig\Extension;
+
+use Symfony\Component\Serializer\SerializerInterface;
+use Twig\Extension\RuntimeExtensionInterface;
+
+/**
+ * @author Jesse Rushlow <jr@rushlow.dev>
+ */
+class SerializeRuntime implements RuntimeExtensionInterface
```

Comment:
should be final, as other runtime classes of the bridge.

Reply:
done